client/proxy/index.js

```diff
@@ -260,7 +260,7 @@ class ZProxy {
                     parsedUrl
                 );
                 contentType = response._contentType;
-            } else if (config.get('mode') === 'e2e') {
+            } else if (config.get('mode') === 'e2e' || config.get('mode') === 'zappdev') {
                 // when MODE=e2e is set this site will be loaded directly from the local system - useful for Zapp developers :)
```

Comment:
@jensendarren with this option fixed for zappdev env when we access the first page of point the example zapp presents their front-end, but they are not deployed to arweave and ganache. This may lead to confusion to zapp developers. Without this option a 404 is presented, which is a better way to say you need to deploy this Zapp. Furthermore, not all zapps will be in example folder, so this may crash with new zapps developed. For this reason I removed this option. The zapps loading is still super fast without it.